app/src/main/java/ltd/evilcorp/atox/activity/ChatActivity.kt

```diff
@@ -17,15 +18,25 @@ class ChatActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_chat)
 
-        messages.add(MessageModel("hello", Sender.Received))
-        messages.add(MessageModel("how are you", Sender.Received))
-        messages.add(MessageModel("I'm good, thanks.", Sender.Sent))
-
         val adapter = MessagesAdapter(this, messages)
         val messageView = findViewById<ListView>(R.id.messages)
         messageView.adapter = adapter
 
+        val handler = Handler(Looper.getMainLooper()) {
```

Comment:
Forgot to delete something you were testing?

Reply:
done